keyboards/handwired/prime_exl/info.json

```diff
@@ -0,0 +1,12 @@
":0,"rx":6.25,"ry":0,"w":1,"h":1}, {"x":19.049999999999997,"y":0.46,"r":0,"rx":6.25,"ry":0,"w":2,"h":1}, {"x":6.96,"y":1.47,"r":0,"rx":6.25,"ry":0,"w":1.26,"h":1}, {"x":8.21,"y":1.47,"r":0,"rx":6.25,"ry":0,"w":1,"h":1}, {"x":18.44,"y":1.47,"r":0,"rx":6.25,"ry":0,"w":1,"h":1}, {"x":19.44,"y":1.47,"r":0,"rx":6.25,"ry":0,"w":1.75,"h":1}, {"x":6.8100000000000005,"y":2.4799999999999995,"r":0,"rx":6.25,"ry":0,"w":1.75,"h":1}, {"x":8.56,"y":2.4799999999999995,"r":0,"rx":6.25,"ry":0,"w":1,"h":1}, {"x":18.08,"y":2.4799999999999995,"r":0,"rx":6.25,"ry":0,"w":1,"h":1}, {"x":19.08,"y":2.4799999999999995,"r":0,"rx":6.25,"ry":0,"w":1,"h":1}, {"x":20.08,"y":2.4799999999999995,"r":0,"rx":6.25,"ry":0,"w":1.25,"h":1}, {"x":6.92,"y":3.4899999999999993,"r":0,"rx":6.25,"ry":0,"w":1.25,"h":1}, {"x":8.17,"y":3.4899999999999993,"r":0,"rx":6.25,"ry":0,"w":1.25,"h":1}, {"x":18.22,"y":3.4899999999999993,"r":0,"rx":6.25,"ry":0,"w":1.5,"h":1}, {"x":19.72,"y":3.4899999999999993,"r":0,"rx":6.25,"ry":0,"w":1.5,"h":1}, {"x":9.25,"y":-8.881784197001252e-16,"r":8,"rx":6.25,"ry":0,"w":1,"h":1}, {"x":10.25,"y":-8.881784197001252e-16,"r":8,"rx":6.25,"ry":0,"w":1,"h":1}, {"x":11.25,"y":-8.881784197001252e-16,"r":8,"rx":6.25,"ry":0,"w":1,"h":1}, {"x":12.25,"y":-8.881784197001252e-16,"r":8,"rx":6.25,"ry":0,"w":1,"h":1}, {"x":9.5,"y":0.9999999999999991,"r":8,"rx":6.25,"ry":0,"w":1,"h":1}, {"x":10.5,"y":0.9999999999999991,"r":8,"rx":6.25,"ry":0,"w":1,"h":1}, {"x":11.5,"y":0.9999999999999991,"r":8,"rx":6.25,"ry":0,"w":1,"h":1}, {"x":12.5,"y":0.9999999999999991,"r":8,"rx":6.25,"ry":0,"w":1,"h":1}, {"x":10,"y":1.9999999999999991,"r":8,"rx":6.25,"ry":0,"w":1,"h":1}, {"x":11,"y":1.9999999999999991,"r":8,"rx":6.25,"ry":0,"w":1,"h":1}, {"x":12,"y":1.9999999999999991,"r":8,"rx":6.25,"ry":0,"w":1,"h":1}, {"x":13,"y":1.9999999999999991,"r":8,"rx":6.25,"ry":0,"w":1,"h":1}, {"x":10,"y":2.999999999999999,"r":8,"rx":6.25,"ry":0,"w":1,"h":1}, {"x":11,"y":2.999999999999999,"r":8,"rx":6.25,"ry":0,"w":1,"h":1}, {"x":12,"y":2.999999999999999,"r":8,"rx":6.25,"ry":0,"w":2,"h":1}, {"x":13.75,"y":2.099999999999999,"r":-8,"rx":6.25,"ry":0,"w":1,"h":1}, {"x":14.75,"y":2.099999999999999,"r":-8,"rx":6.25,"ry":0,"w":1,"h":1}, {"x":15.75,"y":2.099999999999999,"r":-8,"rx":6.25,"ry":0,"w":1,"h":1}, {"x":16.75,"y":2.099999999999999,"r":-8,"rx":6.25,"ry":0,"w":1,"h":1}, {"x":14,"y":3.099999999999999,"r":-8,"rx":6.25,"ry":0,"w":1,"h":1}, {"x":15,"y":3.099999999999999,"r":-8,"rx":6.25,"ry":0,"w":1,"h":1}, {"x":16,"y":3.099999999999999,"r":-8,"rx":6.25,"ry":0,"w":1,"h":1}, {"x":17,"y":3.099999999999999,"r":-8,"rx":6.25,"ry":0,"w":1,"h":1}, {"x":13.5,"y":4.1,"r":-8,"rx":6.25,"ry":0,"w":1,"h":1}, {"x":14.5,"y":4.1,"r":-8,"rx":6.25,"ry":0,"w":1,"h":1}, {"x":15.5,"y":4.1,"r":-8,"rx":6.25,"ry":0,"w":1,"h":1}, {"x":16.5,"y":4.1,"r":-8,"rx":6.25,"ry":0,"w":1,"h":1}, {"x":13.5,"y":5.1,"r":-8,"rx":6.25,"ry":0,"w":2,"h":1}, {"x":15.5,"y":5.1,"r":-8,"rx":6.25,"ry":0,"w":1,"h":1}, {"x":16.5,"y":5.1,"r":-8,"rx":6.25,"ry":0,"w":1,"h":1}]
```

Comment:
Okay, this version is valid JSON but you've run into a bug of sorts; not your fault.

The QMK docs on `info.json` say that the Configurator understands rotated keys, but it actually doesn't. If you preview this file in the Configurator you'll see what I mean. You should set up your layout in KLE as if none of the keys are rotated. So Num Lock, Home, End, Numpad Slash, Numpad Asterisk, Backspace, Escape, Q, W, etc., all in one row with the same `y` value, if we're going by [the image from the readme file](https://i.imgur.com/3t524g8.jpg).

Additionally, the number of objects in the `layouts/LAYOUT/layout` branch of the JSON tree needs to match the number of arguments given for the `LAYOUT` macro in `prime_exl.h`. You have 73 objects in the JSON branch, but the keyboard has 77 keys. The Configurator won't know how to match the rendered key with its argument in the `LAYOUT` macro and also won't render the keyboard properly with this mismatch.

The `prime_exl.h` and `keymap.c` files should be fixed first though, because those changes affect how this file works.

Reply:
yes I noticed the missing keys in the preview after you gave me the tip on how to view it in Configurator. I'll get these ironed out as well as making the keymap.c and prime_exl.h easier to understand which keys should be edited. 

thank you for your patience as I stumble my way through this first submission.

Reply:
Not a problem. There's a bit of a learning curve, but once you understand what's going on, it's smooth sailing.